lib/rules/indent.js

```diff
@@ -53,7 +56,8 @@ module.exports = function(context) {
 
         if (context.options[1]) {
             var opts = context.options[1];
-            options.indentSwitchCase = opts.indentSwitchCase === true;
+            options.SwitchCase = opts.SwitchCase || 0;
+            options.VariableDeclarator = opts.VariableDeclarator || 1;
```

Comment:
If someone sets `VariableDeclarator` to 0, then this will return 1. You can simply by doing this:

```js
var assign = require("object-assign");
assign(options, opts);
```